pkg/probes/firewall_load_balance.go

```diff
@@ -0,0 +1,156 @@
+package probes
```

Comment:
Go seems to, and I agree with it, prefer having singular for package names.
If you feel strongly about it I can let it slide, but I would prefer singular if possible.